image/v3/modifying/text-fonts.md

```diff
@@ -123,7 +123,7 @@ point. Possible values are top, bottom and middle. Default: `bottom`
 
 ### Rotation
 
-> public FontInterface::angle(float $angle): FontInterface
+> public FontInterface::setAngle(float $angle): FontInterface
```

Comment:
Please revert. This is correct call of class Intervention\Image\Typography\FontFactory::class.